content/non-web-ict.md

```diff
@@ -28,7 +28,9 @@ ref: /standards-guidelines/wcag/non-web-ict/
 
 This page introduces the document: "Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)".
 
-Quick link: [WCAG2ICT Note](https://www.w3.org/TR/wcag2ict/) (from 2013)
+Quick links:
+* [WCAG2ICT Note for WCAG 2.o, 2013](https://www.w3.org/TR/wcag2ict/)
+* [WCAG2ICT Working Draft Note for WCAG 2.2](https://www.w3.org/@@) @@link
```

Comment:
This is TR link
```suggestion
* [WCAG2ICT Working Draft Note for WCAG 2.2](https://www.w3.org/TR/2023/DNOTE-wcag2ict-20230815/)
```

Reply:
My understanding is that the draft we publish will replace the existing Note. Also, in these resources, we use the latest version link (partly because it's simpler for users and partly so it doesn't get stale.)